services/api/src/resolvers.js

```diff
@@ -115,12 +115,12 @@ const {
 } = require('./resources/notification/resolvers');
 
 const {
-  addOpenshift,
-  deleteOpenshift,
+  addKubernetes,
+  deleteKubernetes,
   getAllOpenshifts,
   getOpenshiftByProjectId,
-  updateOpenshift,
-  deleteAllOpenshifts,
+  updateKubernetes,
+  deleteAllKubernetes,
```

Comment:
This should be removed in #2123

Reply:
reverted this one to make merging w/ 2123 easier